spec/services/curator/collection_updater_service_spec.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Curator::CollectionUpdaterService, type: :service do
+  before(:all) do
+    @collection ||= create(:curator_collection)
```

Comment:
Curious about the use of `||=` and `@instance_variable` in these updater specs. Do we actually want objects created in other specs to be used here?

Reply:
Not sure if they get shared between specs if you memonize them this way. wanted to ensure the data generated for this set of specs didn't change in any way.